docs/docsets/UB.docset/Contents/Resources/Documents/undocumented.json

```diff
@@ -1,12 +1,16 @@
 {
   "warnings": [
+<<<<<<< HEAD
     {
       "file": "/Users/deaneigenmann/Documents/ub/UB.swift/Sources/UB/Node.swift",
       "line": 14,
       "symbol": "Node.init()",
       "symbol_kind": "source.lang.swift.decl.function.method.instance",
       "warning": "undocumented"
     }
+=======
+
+>>>>>>> master
```

Comment:
? @tueric, maybe delete docs/ and regenerate?

Reply:
ok